Parsers/ASimUserManagement/Parsers/ASimUserManagementCiscoISE.yaml

```diff
@@ -0,0 +1,60 @@
+    let EventOriginalTypeList = toscalar(EventFieldsLookup 
+        | summarize make_set(EventOriginalType));
+    let CiscoISEUsrMgmtParser=(disabled:bool=false){
+      Syslog
+      | where not(disabled)
+      | where ProcessName has_any ("CISE", "CSCO")
+      | parse kind = regex SyslogMessage with @"\d{10}\s" EventOriginalType @"\s(NOTICE|INFO|WARN|WARNING|ERROR|FATAL|DEBUG)"
```

Comment:
can we avoid regex and use Parse here?

Reply:
we can't avoid using `parse` in regex mode here

Reply:
can we use parse function to extract this value EventOriginalType e.g.

| parse SyslogMessage with * " " longvalue:long " " EventOriginalType:int " " *

Reply:
Perfect solution. Added to the parser